src/Utilities/Routes.js

```diff
@@ -38,38 +40,48 @@ const Reports = asyncComponent(() =>
     import(/* webpackChunkName: "Reports" */ '../Components/SmartComponents/Reports/ReportsPage')
 );
 
-const InsightsRoute = ({ component: Component, rootClass = 'vulnerability', ...rest }) => {
+const InsightsRoute = ({ component: Component, rootClass = 'vulnerability', title, ...rest }) => {
     const root = document.getElementById('root');
     root.removeAttribute('class');
 
     root.classList.add(`page__${rootClass}`, 'pf-l-page__main');
     root.classList.add(`page__${rootClass}`, 'pf-c-page__main');
     root.setAttribute('role', 'main');
 
+    if (title) {
+        const subPath = rest.location.pathname && rest.location.pathname.split('/')[2];
+        document.title = (subPath ? subPath + ' - ' : '') + title + ' - ' + intl.formatMessage(messages.pageTitleSuffix);
```

Comment:
it can be written with nested literal template  
document.title = `${subPath ? `${subPath} -` : ''} ${title} - ${intl.formatMessage(messages.pageTitleSuffix)}`;